packages/vite/src/build/__tests__/nestedPages.test.ts

```diff
@@ -50,11 +29,13 @@ test('transform', async () => {
 
   const transformed = await transform('Router.jsx')
   expect(transformed).toEqual(
-    '/* @__PURE__ */ React.createElement(Router, null, /* @__PURE__ */ React.createElement(Route, { path: "/", page: HomePage, name: "home" }));\n'
+    '/* @__PURE__ */ React.createElement(Router, null, /* @__PURE__ */ React.createElement(Route, { path: "/", page: HomePage, name: "home" }));\n',
   )
 })
 
-describe('User specified imports, with static imports', () => {
+// TODO(jgmw): Discuss these tests and the expected output, I don't understand exactly what we should be
```

Comment:
I'll come back to this once you're ready with the rest of the PR and validate the output